New material extra inputs + ripple effect + some fixes

maicol07 opened the pull request:
* Added ripple effect on buttons and repo navigation tabs
* Fixed some shadows bugs
* Fixed some corners
* Fixed navbar text position
* Fixed header search bar

maicol07 committed:
Update README.md

maicol07 committed:
Update README.md

maicol07 committed:
Update style.css

* Added ripple effect on buttons and repo navigation tabs
* Fixed some shadows bugs
* Fixed some corners
* Fixed navbar text position
* Fixed header search bar

DVDAndroid commented:
Hey! Thanks for this contribution.

Here's some things I noticed:

![image](https://user-images.githubusercontent.com/6277172/80726351-938f6600-8b04-11ea-8912-f0083e085202.png)
some outlines are overlapping, I think it's good to leave only the blue outline and remove the other ones

![image](https://user-images.githubusercontent.com/6277172/80727021-7ad38000-8b05-11ea-8edd-c68d33ff6b77.png)
this is a bug: I can select other types of merge, but the arrow for the drop menu disappeared

Correct one: 
![image](https://user-images.githubusercontent.com/6277172/80727264-c423cf80-8b05-11ea-966a-4e635200d315.png)

DVDAndroid commented:
![image](https://user-images.githubusercontent.com/6277172/80757352-5347dc00-8b34-11ea-9d20-2364cead8073.png)
Also there's no shadow of dialogs

maicol07 committed:
Update style.css

* Added ripple to navbar links and dropdowns
* Tweaked border and its radius of boxes
* Fixed repo navbar tabs shadow
* Fixed bouncing notification indicator dimensions
* Fixed other shadow-related things...
* Fixed margin of PR navbar

maicol07 commented:
Update:
* Added ripple to navbar links and dropdowns
* Tweaked border and its radius of boxes
* Fixed repo navbar tabs shadow
* Fixed bouncing notification indicator dimensions
* Fixed other shadow-related things...
* Fixed margin of PR navbar

@DVDAndroid your issues should be fixed

DVDAndroid commented:
> ![image](https://user-images.githubusercontent.com/6277172/80757352-5347dc00-8b34-11ea-9d20-2364cead8073.png)
> Also there's no shadow of dialogs

mhh
This issue is still persists; I'll try to "debug" and understand what's broken

maicol07 commented:
Strange... are you sure you updated the code?
![immagine](https://user-images.githubusercontent.com/9463142/80914501-469dd080-8d4c-11ea-87dc-5a25f48e8cda.png)

DVDAndroid commented:
yep

DVDAndroid commented:
wait wait
I'm saying there's no shadow below the alert dialog

![image](https://user-images.githubusercontent.com/6277172/80915026-acd82280-8d4f-11ea-874e-2f31c95f6066.png)

maicol07 commented:
Yeah, you're right. I'll see how to fix this as I have no idea of what element has the shadow class

DVDAndroid commented:
The only difference I notice it's a `::after` element added after I toggle the style

![image](https://user-images.githubusercontent.com/6277172/80915513-1279de00-8d53-11ea-9507-3316f9c0f9ab.png)

maicol07 commented:
No, that's the button ripple effect. The background "blur" should be applied to the body...

maicol07 committed:
New extra inputs

* Includes fix for dialogs shadows
* Some code reformat

maicol07 committed:
New extra inputs

maicol07 committed:
Update README.md

Added extra features instructions

maicol07 commented:
I think this is all

DVDAndroid commented:
Shadow issue fixed!!

but, why the background behind the input text field is grey?

![image](https://user-images.githubusercontent.com/6277172/81853015-2fd15800-955c-11ea-904f-2034e4da33d9.png)
![image](https://user-images.githubusercontent.com/6277172/81853069-437cbe80-955c-11ea-905b-924e0ff13d3a.png)


**maybe [this](https://github.com/DVDAndroid/GitHubMaterialDesign/blob/529b9d9612fbd429d636d94c16ea3f2908c37dc1/style.css#L460)?**

maicol07 commented:
That's new... I'll check it later

DVDAndroid commented:
One last thing... :/ 

![image](https://user-images.githubusercontent.com/6277172/81853985-855a3480-955d-11ea-9d07-6c06288a53d3.png)

(on pull requests)

DVDAndroid commented:
Inoltre, puoi aggiungerti ai crediti nel readme.md; nessun problema

maicol07 committed:
Update README.md

Updated credits

maicol07 committed:
Update style.css

* Fixes

maicol07 committed:
Update githubmd.user.js

maicol07 commented:
Ok, dovrebbe essere tutto risolto

DVDAndroid commented:
Ok mi sembra tutto funzionante
solamente non riesco a vedere differenze con l'userscript; ho capito che aggiunge una classe che implementa una animazione, ma non vedo cambiamenti in caselle di testo, casella di ricerca ecc...

maicol07 commented:
Rende il bordo nero (avevo anche impostato l'animazione dell'etichetta che si univa al bordo, ma dava problemi di visualizzazione in alcuni casi e quindi l'ho tolta. Se vuoi puoi provare a migliorarla, il link dello script è nel README):
![immagine](https://user-images.githubusercontent.com/9463142/81921925-9f3a5c80-95db-11ea-9ac3-225c0e9d910d.png)
L'idea iniziale comunque era di inserire la casella di testo del material design attuale (filled o outline).

DVDAndroid commented:
mhhh ok, non trovo cambiamenti; 😢  nessun errore nella console di chrome

![image](https://user-images.githubusercontent.com/6277172/81922653-b9286f00-95dc-11ea-8ebd-c07c8ae768a5.png)

se è possibile sistemare ok, dopo comunque mergerò la pr

DVDAndroid commented:
altro problema: 
![image](https://user-images.githubusercontent.com/6277172/82063075-51048680-96cb-11ea-91bb-9a1886d79a71.png)

maicol07 committed:
Update style.css

* Fixed some menus

maicol07 commented:
Risolto!

DVDAndroid committed:
Update style.css

DVDAndroid commented:
Ora lo carico su userstyles

Grazie!
